firmware/README.md

```diff
@@ -11,6 +11,41 @@ Right click project > `Build Configurations` > `Set Active` > your platform.
 
 ## Design Spec
 
+### Revision 1
+
+#### Playtest Notes
+The following notes are for the specification designed below, implemented as of [firmware_rev1](https://github.com/Gigahawk/mech423_final_project/tree/firmware_rev1/firmware) (commit hash 97b537f)
+
+- Timing is both too slow and too fast
+    - Movement of ball between handles takes too long, but the window to return a ball is too small, no good middle ground exists
+        - Solution(s):
+            - Player should be able to return the ball when the ball reaches a few LEDs prior to the last one (button should light up a few LEDs earlier)
```

Comment:
Maybe as a subitem to this one; adjusting the starting speed to be faster just because it would still be way too slow at the beginning

Reply:
That's implicit, making the return region bigger gives greater flexibility to play with the numbers, but I'll add that in.